<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogNego</class>
 <widget class="QDialog" name="dialogNego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nego</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Origen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineOrigen"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Destino</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineDestino"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>NÂº Plazas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Fecha</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonOkCancel">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>dialogNego</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>dialogNego</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
